xbmc/video/windows/GUIWindowVideoBase.cpp

```diff
@@ -1635,11 +1635,11 @@ bool CGUIWindowVideoBase::GetDirectory(const std::string &strDirectory, CFileIte
     newPlaylist->m_bIsFolder = true;
     items.Add(newPlaylist);
 
-/*    newPlaylist.reset(new CFileItem("newplaylist://", false));
```

Comment:
Are you sure that the playlist implementation works properly now? Either way IMO this should go into a separate commit and PR.